BitwardenShared/UI/Auth/ProfileSwitcher/ProfileSwitcherItem.swift

```diff
@@ -5,6 +5,9 @@ import SwiftUI
 /// An object that defines account profile information relevant to account switching
 /// Part of `ProfileSwitcherState`.
 struct ProfileSwitcherItem: Equatable, Hashable {
+    /// Indicates if the account can be locked.
+    var canBeLocked: Bool
```

Comment:
🤔 There is also an accessibility action in `ProfileSwitcherRow` which I think needs to be gated behind this.

https://github.com/bitwarden/ios/blob/bd6651bc1a222271549f6f2f9221eff9b3b9d799/BitwardenShared/UI/Auth/ProfileSwitcher/ProfileSwitcherRow.swift#L229-L234